packages/client/README.md

```diff
@@ -114,19 +116,16 @@ const response = await client.getCsv({
 ```
 
 ```text
-"Count_Person_BelowPovertyLevelInThePast12Months.date","Count_Person_BelowPovertyLevelInThePast12Months.name","Count_Person_BelowPovertyLevelInThePast12Months.unit","Count_Person_BelowPovertyLevelInThePast12Months.unitDisplayName","Count_Person_BelowPovertyLevelInThePast12Months.value","entity.isoCode","entity.name","entity.dcid"
-"2021","Population Below Poverty Level Status in Past Year",null,null,769819,"US-AL","Alabama","geoId/01"
-"2021","Population Below Poverty Level Status in Past Year",null,null,75016,"US-AK","Alaska","geoId/02"
-"2021","Population Below Poverty Level Status in Past Year",null,null,934911,"US-AZ","Arizona","geoId/04"
-"2021","Population Below Poverty Level Status in Past Year",null,null,468113,"US-AR","Arkansas","geoId/05"
-"2021","Population Below Poverty Level Status in Past Year",null,null,4741175,"US-CA","California","geoId/06"
-"2021","Population Below Poverty Level Status in Past Year",null,null,535976,"US-CO","Colorado","geoId/08"
-"2021","Population Below Poverty Level Status in Past Year",null,null,351476,"US-CT","Connecticut","geoId/09"
-"2021","Population Below Poverty Level Status in Past Year",null,null,109274,"US-DE","Delaware","geoId/10"
-"2021","Population Below Poverty Level Status in Past Year",null,null,100618,"US-DC","District of Columbia","geoId/11"
-"2021","Population Below Poverty Level Status in Past Year",null,null,2744612,"US-FL","Florida","geoId/12"
-"2021","Population Below Poverty Level Status in Past Year",null,null,1441351,"US-GA","Georgia","geoId/13"
-"2021","Population Below Poverty Level Status in Past Year",null,null,133740,"US-HI","Hawaii","geoId/15"
+"entity.dcid","entity.properties.isoCode","entity.properties.name","variable.dcid","variable.observation.date","variable.observation.value","variable.properties.name"
+"geoId/01","US-AL","Alabama","Count_Person_BelowPovertyLevelInThePast12Months","2022",768897,"Population Below Poverty Level Status in Past Year"
+"geoId/02","US-AK","Alaska","Count_Person_BelowPovertyLevelInThePast12Months","2022",75227,"Population Below Poverty Level Status in Past Year"
+"geoId/04","US-AZ","Arizona","Count_Person_BelowPovertyLevelInThePast12Months","2022",916876,"Population Below Poverty Level Status in Past Year"
+"geoId/05","US-AR","Arkansas","Count_Person_BelowPovertyLevelInThePast12Months","2022",475729,"Population Below Poverty Level Status in Past Year"
+"geoId/06","US-CA","California","Count_Person_BelowPovertyLevelInThePast12Months","2022",4685272,"Population Below Poverty Level Status in Past Year"
+"geoId/08","US-CO","Colorado","Count_Person_BelowPovertyLevelInThePast12Months","2022",540105,"Population Below Poverty Level Status in Past Year"
+"geoId/09","US-CT","Connecticut","Count_Person_BelowPovertyLevelInThePast12Months","2022",355692,"Population Below Poverty Level Status in Past Year"
```

Comment:
not sure if this PR is the best place for this discussion, but do the getCSV functions wrap each string in quotes? What do you think of removing the quotes, and having the user infer type? Removing quotes may be easier when parsing downstream / loading into spreadsheets, etc.

Reply:
it does wrap each string in quotes, which is a bit verbose, but could be important for the "date" column. Dates are stored as strings, and can be values like YYYY (e.g, "2022") or YYYY-MM-DD "2022-01-15". I like the explicitness of saying dates are strings in this case to avoid any parsing erors.